### PR TITLE
Test 2 - Dismissals.

### PR DESCRIPTION
**Pull request by marchermans**

### CODE REVIEW

File: src/main/resources/mcmod.info [hunk 1]
```diff
@@ -2,7 +2,7 @@
 {
   "modid": "minecolonies",
```
Comment (LDTTeam-Buildsystem) [inline src/main/resources/mcmod.info:3]: ### Infos:
- Typo: In word 'minecolonies'
- Typo: In word 'modid'

**Review by LDTTeam-Buildsystem — DISMISSED** [review]
#### Analysis Complete
##### Statistics:
|        | Count |  New  | Old |  Diff  |
| ------ |:-----:|:-----:|:---:|:------:|
| Total  | 29854 | 29192 |  0  | +29192 |
| Errors |   8   |   0   |  0  |   +0   |
##### More Information:
- [PR](https://teamcity.minecolonies.com/viewLog.html?buildId=125&tab=Inspection&buildTypeId=bt2)

File: src/main/resources/mcmod.info [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
+  "description": " MineColonies is a Colony/Town Simulator that adds many structures and NPC workers and guards that you control and put to work for you! Based around a townhall and village mechanic, there are numerous workers that can be added to your colony, building a busy and vibrant village/town/city within Minecraft. There is no coded limit to the number of NPC's at your disposal, so you can create a small efficient Village dedicated to a particular type of worker, or a sprawling metropolis with many workers. \n It's entirely up to you! This is a new typo: And one more test wiht authorization this time :D ikuahsdiuashdihaiudha asiudhaihsdiasd :D",
```
Comment (LDTTeam-Buildsystem) [inline src/main/resources/mcmod.info]: ### Infos:
- Typo: In word 'ikuahsdiuashdihaiudha'
- Typo: In word 'Minecraft'
- Typo: In word 'wiht'
- Typo: In word 'NPC's'
- Typo: In word 'asiudhaihsdiasd'
- Typo: In word 'townhall'

File: src/main/resources/mcmod.info [hunk 3]
```diff
@@ -2,7 +2,7 @@
...
   "version": "${version}",
   "mcversion": "${mcversion}",
```
Comment (LDTTeam-Buildsystem) [inline src/main/resources/mcmod.info]: ### Infos:
- Typo: In word 'mcversion'

**Review by LDTTeam-Buildsystem — DISMISSED** [review]
#### Analysis Complete
##### Statistics:
|        | Count | New | Old | Diff |
| ------ |:-----:|:---:|:---:|:----:|
| Total  | 29851 |  0  |  3  |  -3  |
| Errors |   8   |  0  |  0  |  0   |
##### More Information:

**Comment by Raycoms** [comment]
:shipit:

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/2573/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

**Comment by Asherslab** [comment]
:shipit:

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/2573/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

File: src/main/resources/mcmod.info (hunk 1 above)
Comment (LDTTeam-Buildsystem) [inline src/main/resources/mcmod.info]: ### Infos:
- Typo: In word 'minecolonies'
- Typo: In word 'modid'

File: src/main/resources/mcmod.info [hunk 4]
```diff
@@ -2,7 +2,7 @@
 {
   "modid": "minecolonies",
   "name": "MineColonies",
-  "description": " MineColonies is a Colony/Town Simulator that adds many structures and NPC workers and guards that you control and put to work for you! Based around a townhall and village mechanic, there are numerous workers that can be added to your colony, building a busy and vibrant village/town/city within Minecraft. There is no coded limit to the number of NPC's at your disposal, so you can create a small efficient Village dedicated to a particular type of worker, or a sprawling metropolis with many workers. \n It's entirely up to you!",
+  "description": " MineColonies is a Colony/Town Simulator that adds many structures and NPC workers and guards that you control and put to work for you! Based around a townhall and village mechanic, there are numerous workers that can be added to your colony, building a busy and vibrant village/town/city within Minecraft. There is no coded limit to the number of NPC's at your disposal, so you can create a small efficient Village dedicated to a particular type of worker, or a sprawling metropolis with many workers. \n It's entirely up to you! Nearing completion iasuhdashiudsh",
```
Comment (LDTTeam-Buildsystem) [inline src/main/resources/mcmod.info]: ### Infos:
- Typo: In word 'NPC's'
- Typo: In word 'Minecraft'
- Typo: In word 'iasuhdashiudsh'
- Typo: In word 'townhall'

File: src/main/resources/mcmod.info [hunk 5]
```diff
@@ -2,7 +2,7 @@
...
   "version": "${version}",
   "mcversion": "${mcversion}",
```
Comment (LDTTeam-Buildsystem) [inline src/main/resources/mcmod.info]: ### Infos:
- Typo: In word 'mcversion'

File: src/main/resources/mcmod.info (hunk 1 above)
Comment (LDTTeam-Buildsystem) [inline src/main/resources/mcmod.info]: ### Infos:
- Typo: In word 'minecolonies'
- Typo: In word 'modid'

File: src/main/resources/mcmod.info [hunk 6]
```diff
@@ -2,7 +2,7 @@
...
   "modid": "minecolonies",
   "name": "MineColonies",
-  "description": " MineColonies is a Colony/Town Simulator that adds many structures and NPC workers and guards that you control and put to work for you! Based around a townhall and village mechanic, there are numerous workers that can be added to your colony, building a busy and vibrant village/town/city within Minecraft. There is no coded limit to the number of NPC's at your disposal, so you can create a small efficient Village dedicated to a particular type of worker, or a sprawling metropolis with many workers. \n It's entirely up to you!",
+  "description": " MineColonies is a Colony/Town Simulator that adds many structures and NPC workers and guards that you control and put to work for you! Based around a townhall and village mechanic, there are numerous workers that can be added to your colony, building a busy and vibrant village/town/city within Minecraft. There is no coded limit to the number of NPC's at your disposal, so you can create a small efficient Village dedicated to a particular type of worker, or a sprawling metropolis with many workers. \n It's entirely up to you! Trying with deletion? iaushdiaushdihas",
```
Comment (LDTTeam-Buildsystem) [inline src/main/resources/mcmod.info]: ### Infos:
- Typo: In word 'townhall'
- Typo: In word 'Minecraft'
- Typo: In word 'iaushdiaushdihas'
- Typo: In word 'NPC's'

File: src/main/resources/mcmod.info [hunk 7]
```diff
@@ -2,7 +2,7 @@
...
   "version": "${version}",
   "mcversion": "${mcversion}",
```
Comment (LDTTeam-Buildsystem) [inline src/main/resources/mcmod.info]: ### Infos:
- Typo: In word 'mcversion'

File: src/main/resources/mcmod.info (hunk 1 above)
Comment (LDTTeam-Buildsystem) [inline src/main/resources/mcmod.info]: ### Infos:
- Typo: In word 'minecolonies'
- Typo: In word 'modid'

File: src/main/resources/mcmod.info [hunk 8]
```diff
@@ -2,7 +2,7 @@
...
-  "description": " MineColonies is a Colony/Town Simulator that adds many structures and NPC workers and guards that you control and put to work for you! Based around a townhall and village mechanic, there are numerous workers that can be added to your colony, building a busy and vibrant village/town/city within Minecraft. There is no coded limit to the number of NPC's at your disposal, so you can create a small efficient Village dedicated to a particular type of worker, or a sprawling metropolis with many workers. \n It's entirely up to you!",
+  "description": " MineColonies is a Colony/Town Simulator that adds many structures and NPC workers and guards that you control and put to work for you! Based around a townhall and village mechanic, there are numerous workers that can be added to your colony, building a busy and vibrant village/town/city within Minecraft. There is no coded limit to the number of NPC's at your disposal, so you can create a small efficient Village dedicated to a particular type of worker, or a sprawling metropolis with many workers. \n It's entirely up to you! Trying with nope. Then the comment deletion maybe? sdfhgdfj xvbnv m wsrteuz ",
```
Comment (LDTTeam-Buildsystem) [inline src/main/resources/mcmod.info]: ### Infos:
- Typo: In word 'NPC's'
- Typo: In word 'xvbnv'
- Typo: In word 'sdfhgdfj'
- Typo: In word 'Minecraft'
- Typo: In word 'wsrteuz'
- Typo: In word 'townhall'

File: src/main/resources/mcmod.info [hunk 9]
```diff
@@ -2,7 +2,7 @@
...
   "version": "${version}",
   "mcversion": "${mcversion}",
```
Comment (LDTTeam-Buildsystem) [inline src/main/resources/mcmod.info]: ### Infos:
- Typo: In word 'mcversion'

File: src/main/resources/mcmod.info (hunk 1 above)
Comment (LDTTeam-Buildsystem) [inline src/main/resources/mcmod.info]: ### Infos:
- Typo: In word 'minecolonies'
- Typo: In word 'modid'

File: src/main/resources/mcmod.info [hunk 10]
```diff
@@ -2,7 +2,7 @@
...
   "name": "MineColonies",
-  "description": " MineColonies is a Colony/Town Simulator that adds many structures and NPC workers and guards that you control and put to work for you! Based around a townhall and village mechanic, there are numerous workers that can be added to your colony, building a busy and vibrant village/town/city within Minecraft. There is no coded limit to the number of NPC's at your disposal, so you can create a small efficient Village dedicated to a particular type of worker, or a sprawling metropolis with many workers. \n It's entirely up to you!",
+  "description": " MineColonies is a Colony/Town Simulator that adds many structures and NPC workers and guards that you control and put to work for you! Based around a townhall and village mechanic, there are numerous workers that can be added to your colony, building a busy and vibrant village/town/city within Minecraft. There is no coded limit to the number of NPC's at your disposal, so you can create a small efficient Village dedicated to a particular type of worker, or a sprawling metropolis with many workers. \n It's entirely up to you! iuaszdiuazdia iaszdiazidz iasuzdiauzsd One more test.",
```
Comment (LDTTeam-Buildsystem) [inline src/main/resources/mcmod.info]: ### Infos:
- Typo: In word 'iuaszdiuazdia'
- Typo: In word 'NPC's'
- Typo: In word 'iasuzdiauzsd'
- Typo: In word 'Minecraft'
- Typo: In word 'townhall'
- Typo: In word 'iaszdiazidz'

File: src/main/resources/mcmod.info [hunk 11]
```diff
@@ -2,7 +2,7 @@
...
   "version": "${version}",
   "mcversion": "${mcversion}",
```
Comment (LDTTeam-Buildsystem) [inline src/main/resources/mcmod.info]: ### Infos:
- Typo: In word 'mcversion'

File: src/main/resources/mcmod.info (hunk 1 above)
Comment (LDTTeam-Buildsystem) [inline src/main/resources/mcmod.info]: ### Infos:
- Typo: In word 'minecolonies'
- Typo: In word 'modid'

File: src/main/resources/mcmod.info [hunk 12]
```diff
@@ -2,7 +2,7 @@
...
-  "description": " MineColonies is a Colony/Town Simulator that adds many structures and NPC workers and guards that you control and put to work for you! Based around a townhall and village mechanic, there are numerous workers that can be added to your colony, building a busy and vibrant village/town/city within Minecraft. There is no coded limit to the number of NPC's at your disposal, so you can create a small efficient Village dedicated to a particular type of worker, or a sprawling metropolis with many workers. \n It's entirely up to you!",
+  "description": " MineColonies is a Colony/Town Simulator that adds many structures and NPC workers and guards that you control and put to work for you! Based around a townhall and village mechanic, there are numerous workers that can be added to your colony, building a busy and vibrant village/town/city within Minecraft. There is no coded limit to the number of NPC's at your disposal, so you can create a small efficient Village dedicated to a particular type of worker, or a sprawling metropolis with many workers. \n It's entirely up to you! Attempting this with a set VCS Root else this will simply not work oisdhfoshfdiushfis",
```
Comment (LDTTeam-Buildsystem) [inline src/main/resources/mcmod.info]: ### Infos:
- Typo: In word 'oisdhfoshfdiushfis'
- Typo: In word 'NPC's'
- Typo: In word 'Minecraft'
- Typo: In word 'townhall'

File: src/main/resources/mcmod.info [hunk 13]
```diff
@@ -2,7 +2,7 @@
...
   "version": "${version}",
   "mcversion": "${mcversion}",
```
Comment (LDTTeam-Buildsystem) [inline src/main/resources/mcmod.info]: ### Infos:
- Typo: In word 'mcversion'

**Review by LDTTeam-Buildsystem — APPROVED** [review]
#### Analysis Complete
##### Statistics:
|        | Count | New | Old | Diff |
| ------ |:-----:|:---:|:---:|:----:|
| Total  |   0   |  0  |  0  |  0   |
| Errors |   0   |  0  |  0  |  0   |
##### More Information:

**Pull request closed without merging**